model/form-section.js

```diff
@@ -313,6 +313,18 @@ module.exports = memoize(function (db) {
 
 				return result;
 			}
+		},
+		toWSSchema: {
+			value: function (ignore) {
+				var schema = {};
+				this.propertyNames.forEach(function (prop) {
```

Comment:
Maybe we should be explicit about operating on prototype in order to avoid any leaks from instance. So basically make sure that we operate on respective prototype in scope of sections.